lib_tflite_micro/src/tflite-xcore-kernels/xcore_load_from_flash_v2.cc

```diff
@@ -85,8 +85,8 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
 #else
   // TODO: debug this
   FILE *fd = fopen("flash.bin", "rb");
```

Comment:
it the flash file intended to be hardcoded here?

Reply:
This is under work. @henkmuller will update it to read from memory instead of a file on x86.